Merged Tool Homepage Links in config.js

GeekBoySupreme opened the pull request:
#383 

This PR merges the Tool Homepages from tool-homepages.ts to config.js.
Both as an Array and individual variables.

GeekBoySupreme committed:
Merge pull request #2 from GoogleChromeLabs/dev

Syncing Commits with Base Repository

GeekBoySupreme committed:
Getting Tool Homepages to config.js

GeekBoySupreme commented:
@googlebot I signed it!

GeekBoySupreme committed:
Restructuring testSubjects to include homepage link

surma commented:
@GeekBoySupreme pretty sure this PR breaks the build as it currently is as we use `testSubjects` in a bunch of places throughout the code base. These would need to get adjusted.

GeekBoySupreme commented:
Yup, did not get time to work on this yesterday. Have fixed a couple of instances where testSubjects to fix the build. 
Would be completing that soon.

surma commented:
Okay cool! No rush :)

GeekBoySupreme commented:
Roger 🙌

GeekBoySupreme committed:
Resolving config change issues

GeekBoySupreme commented:
@surma pushed a change and in the config file itself.
Renamed the List to `testSubjectsList` and added this bit of code to generate `testSubjects` -
`export const testSubjects = Object.keys(testSubjectsList).map(key => testSubjectsList[key].name);`

Changing the values to fix `testSubjects` across all the files was not a really good fix. Hence, implemented this.
